src/components/progress-bar/progress-bar.jsx

```diff
@@ -15,36 +17,58 @@ import formatNumber from 'utils/format-number.js';
 
 const style = createStyler(styles, 'progress-bar');
 
+function toPercentage(num, total) {
+  return `${Math.min(Math.max(num / total * 100, 0), 100).toFixed(3)}%`;
```

Comment:
Just out of curiousity, does `num / total` actually leave the [0, 1] range sometimes? :smile:

Reply:
After I added [this check in the code](
https://github.com/mongodb-js/compass-import-export/pull/243/files#diff-263b26f34c94be322bff96c7d050007bdbf56383fa237c82b88c6595c73bd145R312-R319) I don't think it ever will, but my first pass on implementing the change would look like this sometimes (the "guesstimation" there is not very precise)

<img width="400" alt="image" src="https://user-images.githubusercontent.com/5036933/111989440-0d13b680-8b12-11eb-982b-45d67471fd4f.png">


... so I decided to be overly cautious here just in case I missed some weird case 😆